src/app/views/apps/modals/app-connect-modal-cloud.controller.coffee

```diff
@@ -0,0 +1,57 @@
+angular.module 'mnoEnterpriseAngular'
+  .controller('DashboardAppConnectCloudModalCtrl', ($scope, $uibModalInstance, app, MnoeAppInstances, toastr) ->
+
+    $scope.app = app
+    $scope.model = {}
+    $scope.hasLinked = false
+    $scope.hasChosenEntities = false
+    $scope.historicalData = false
+    $scope.isFormLoading = true
+    $scope.isSubmitting = false
+
+    MnoeAppInstances.getForm(app)
+      .then((response) ->
+        $scope.schema = response.schema
+        $scope.form = [ "*" ]
+        $scope.isFormLoading = false
+      )
+
+    $scope.getDate = ->
+      moment.locale('en')
+      moment().format("dddd, MMMM Do YYYY, h:mm:ss a")
```

Comment:
Do not use directly the moment object. Use the angular-moment service wrapper. (cf. https://github.com/urish/angular-moment)

Reply:
For the format see the available i18n'ed format in moment: https://momentjs.com/docs/#/displaying/format/